includes/ServiceWiring.php

```diff
@@ -53,7 +57,52 @@
 		return new MinimalItemInput(
 			$repo->getPropertyDataTypeLookup(),
 			$repo->getValueParserFactory(),
-			WikibaseReconcileEditServices::getReconciliationService( $services )
+			WikibaseReconcileEditServices::getReconciliationService( $services ),
+			WikibaseReconcileEditServices::getPropertyLabelResolver( $services ),
+		);
+	},
+
+	'WikibaseReconcileEdit.PropertyLabelResolver' => function ( MediaWikiServices $services ):
+		CachedDatabasePropertyLabelResolver {
+		$repo = WikibaseRepo::getDefaultInstance();
+		$settings = $repo->getSettings();
+
+		$languageCode = $services->getContentLanguage()->getCode();
```

Comment:
Maybe this should be a setting for the WikibaseReconcileEdit extension rather than defaulting to the language of the mediawiki installation?